aiosmtpd/docs/smtp.rst

```diff
@@ -101,7 +101,7 @@ override to provide additional responses.
 SMTP API
 ========
 
-.. class:: SMTP(handler, *, data_size_limit=33554432, enable_SMTPUTF8=False, decode_data=False, hostname=None, tls_context=None, require_starttls=False, loop=None)
+.. class:: SMTP(handler, *, data_size_limit=33554432, enable_SMTPUTF8=False, decode_data=False, hostname=None, ident=None, tls_context=None, require_starttls=False, loop=None)
```

Comment:
Should we call this `greeting`?  That seems more inline with RFC 5321.  But I guess it does make sense since it mirrors `__ident__`.

(It's fortunate that these are keyword-only arguments, so order doesn't matter.)